lib/pbkit/pbkit.c

```diff
@@ -34,6 +36,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <stdarg.h>
+#include <assert.h>
```

Comment:
This is included twice, once for `DBG` and once unconditionally

Reply:
Conditional include removed.

Reply:
Keep the `assert` include above - it already existed in master

Reply:
Changes to includes reverted. Fixed.